src/client/transport/local.js

```diff
@@ -208,13 +210,13 @@ const localMasters = new Map();
 export function Local(opts) {
   return transportOpts => {
     let master;
-
-    if (localMasters.has(transportOpts.gameKey) & !opts) {
```

Comment:
See https://github.com/nicolodavis/boardgame.io/issues/552 for the context of why `!opts` was added originally.

Reply:
Still no reason to have `!opts` guard. Should be eventually `!(opts.bots)` with explanation in comment why this is here.

Reply:
@nicolodavis @delucis i would be happy to see your comment to above so i will be able to finish this PR. Thx.

Reply:
Let's change it to `!opts.bots` for now and we can eventually add a `forceCreate / createNewMaster` option later.  Thanks!